docs/sources/panels-visualizations/visualizations/traces/index.md

```diff
@@ -77,7 +77,7 @@ To use this procedure, you need:
 - A Grafana instance
 - A [Tempo data source](ref:tempo-data-source) connected to your Grafana instance
 
-### Add the traces visualization query {#add-the-traces-panel-query}
+### Steps {#add-the-traces-panel-query}
```

Comment:
In this case, I think using Steps as haeding works because there's a task heading, before you begin, and then the steps themselves.